task01.rb

```diff
@@ -0,0 +1,12 @@
+#1. Write a script that returns true if a string contains comparison, e.g. "<23", ">45".
+
+def task01(str)
+	if str.class != String
+		raise "Not a valid format. Need a string"
+	end
+  if str.scan(/\A\s*(<|>|<=|>=|=){1}\s*[1-9]+[0-9]*\s*\z/).empty?
+    false
+  else
+    true
```

Comment:
@MaraEfimova Couple comments here:
1. Specify error class on `raise`.
2. I believe `> = 1` should throw an error, it is returning `false` now.
3. `">1.1"`, `">>>"`, `"aa"`, `""` or `"    "` should also throw an error, but are returning `false` now.